openmetadata-ui/src/main/resources/ui/src/components/SampleDataTable/SampleDataTable.component.tsx

```diff
@@ -151,7 +152,17 @@ const SampleDataTable: FunctionComponent<Props> = ({ sampleData }: Props) => {
           </table>
         ) : (
           <div className="tw-flex tw-justify-center tw-font-medium tw-items-center tw-border tw-border-main tw-rounded-md tw-p-8">
-            No sample data available
+            To view Sample Data, run the Profiler Ingestion. Please refer to
```

Comment:
Can we use `Empty` component here from AntD?

Reply:
Made suggested changes.